src/ORM/Association.php

```diff
@@ -366,6 +374,25 @@ public function source(Table $table = null)
     }
 
     /**
+     * The registry alias of the target table object
+     *
+     * @return string
+     */
+    public function getRegistryAlias()
```

Comment:
Should 'target' be in this method's name? It is the alias of the target table after all.